lister/Program.cs

```diff
@@ -34,4 +37,10 @@ static void Main(string[] args)
             }  
         }
     }
+
+    // I changed the style of the comments. Do you usually just leave some comments of the the 
+    // commit you worked on? For example, the only comments in this code at this very moment are
+    // ones related to this commit I'm pushing now.
+    // Also at this point, I may have resolved all outstanding issues. If that is so, can we do
+    // code review so that I can merge this branch with the master?
```

Comment:
I leave comments that explain "the why" of the code.  The idea is in the future, when someone (maybe you) comes back to read the code they can see _what_ the code does by reading it and the comments explain *why* the code is here.

So your comment on line 20 could be:

    // If the file is over-read a bunch of '?'s are in the output, so we use the
    // length of the file to prevent reading too much of the file.